core/driver/sources/src/main/resources/reference.conf

```diff
@@ -5,12 +5,52 @@ probe {
 
   // `intellij` is for org.virtuslab.ideprobe.config.IntellijConfig - default values below:
   intellij {
+    // `plugins` array contains IntelliJ plugins which should be installed and enabled on IntelliJ run. Here you can add
+    // your IntelliJ plugin for tests (amongst other plugins). There are a few ways to provide a proper entry pointing
+    // to a plugin:
+    // 1. By using a direct URI pointing to the plugin (could be from a website or from a local file). For example:
+    #     { uri = "https://packages.jetbrains.team/maven/p/ij/intellij-dependencies/com/intellij/remoterobot/robot-server-plugin/0.11.15/robot-server-plugin/0.11.15.zip" }
+    // 2. By using default IntelliJ plugins repository (see `probe.resolvers.plugins.repository.uri`) with additional
+    //    id (pluginId), version and (optional) channel parameters which is translated into a URL with a query string.
+    //    For example:
+    #       { id = "org.intellij.scala", version = "2020.3.369", channel = "nightly" }
+    //      will be resolved into following URL with query string (from where ide-probe should download the plugin):
+    //      https://plugins.jetbrains.com/plugin/download?pluginId=org.intellij.scala&version=2020.3.369&channel=nightly
+    //    and without the optional "channel" parameter:
+    #       { id = "org.intellij.scala", version = "2020.3.369" }
+    //      will be resolved into following URL with query string (from where ide-probe should download the plugin):
+    //      https://plugins.jetbrains.com/plugin/download?pluginId=org.intellij.scala&version=2020.3.369
+    // 3. By using a bundled IntelliJ plugin. Such plugin should be bundled in your project as a module/subproject and
```

Comment:
Huh _bundled_ already has a specific meaning for IntelliJ plugins: see Plugins in Settings:

![image](https://user-images.githubusercontent.com/3383210/197849638-2f94b59d-ec32-42a9-9971-8e1f44afbc0a.png)

I wouldn't use this word to avoid confusion

Reply:
Ok, but this is ide-probe's naming convention. These "ways" of resolving Plugins are in fact called "Bundled" in code. What do you suggest?
<img width="925" alt="image" src="https://user-images.githubusercontent.com/49597713/197877568-e8a63dfc-894c-43ab-9d94-33c404cc3582.png">

Reply:
Pls open an issue to rethink this naming. No action to be done on this PR yet